Return proper error when backups are empty for details command (#4968)
alcionai/corso: CHANGELOG.md

## [Unreleased] (beta)

### Changed

> - If we try to see details of an empty backup, it now returns a more helpful error message.

Reviewer: ```suggestion
- Running `backup details` on an empty backup returns a more helpful error message.
```
